tutorials/kotlin-springboot-app-engine/index.md

```diff
@@ -99,16 +99,24 @@ To deploy your application, you will use an App Engine plugin for Maven which si
 is also [available for Gradle](https://cloud.google.com/appengine/docs/standard/java/tools/gradle).
 
 
-1.  Create a file called `app.yaml` in a new folder `src/main/appengine` with the following contents:
-
-        runtime: java
-        env: flex
-        runtime_config:
-          jdk: openjdk8
-
-    By specifying `runtime: java`, the runtime image `gcr.io/google-appenine/openjdk:8` is automatically selected
-    when you deploy a JAR (*.jar) file. The JDK version is also selected using the `jdk` field.
-
+1.
+   _When running on JDK 8:_ 
+   
+   Create a file called app.yaml in a new folder src/main/appengine with the following contents:
+
+      runtime: java
+      env: flex
+      runtime_config:
+      jdk: openjdk8
```

Comment:
You have changed the indentation between the original and your modification. The indentation is significant here.

Reply:
I think it should be correct now